src/app/shared/doc-viewer/docs-api-module-import/docs-api-module-import.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { CopierService } from '../../copier/copier.service';
+
+const EXPORT_NAME_PLACEHOLDER = '{{exportName}}';
+const MODULE_IMPORT_PATH_PLACEHOLDER = '{{moduleImportPath}}';
+const MODULE_IMPORT_TEMPLATE = `import {${EXPORT_NAME_PLACEHOLDER}} from '${MODULE_IMPORT_PATH_PLACEHOLDER}';`;
+
+@Component({
+  selector: '[docs-api-module-import]',
+  template: `
+    <p class="docs-api-module-import">
+      <code>
+        {{ moduleImport }}
+      </code>
+    </p>
+    <button
+      class="docs-api-module-import-copy"
+      aria-label="Copy module import"
+      mat-icon-button
+      (click)="copyModuleImport()"
+      matTooltip="Copy module import">
+      <span class="material-icons">file_copy</span>
+    </button>
```

Comment:
We could add some opacity to the icon as it's a bit too strong in pure black. It should still meet a11y contrast guidelines though (in light and dark modes). 

Angular.io changes the color to a darker color on hover rather than changing opacity. So we should consider following that pattern.

Reply:
I added a button color style, the same way it is applied for `example-viewer` and the global icon color styles which are for H3 and H4.